### PR TITLE
Pull request of Design Teardown project for submission for coding review

### PR DESCRIPTION
**Pull request by aulbytj**
Design Teardown project: 
Let the commits reflect, the HTML structure with the relevant sections:
header
employee section
latest post
pre-launch 
community post
curated post
smashing newsletter
footer.
CSS styling for the entire webpage, other files include the README.md file the stickler files and stylelint.config.js files, gitignore file and an image of the interior page in the images folder.

### CODE REVIEW

**Comment by aulbytj** [comment]
i understand that this project has had some discrepancies with the way the page is laid out but please take note of the image attached for this is how the smashing webpage layout is that the project is based on.

image

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @aulbytj and @kagaramag  
### Good job for this amazing work you have done :clap: However, there are some changes you need to make. 
- Validate your HTML code with the online [validator tool](https://validator.w3.org/#validate_by_upload)
Screenshot from 2020-02-11 10-58-54
- Your content overlaps to the right, make the right and left margin be equal and avoid the horizontal scrollbar
Screenshot from 2020-02-11 11-00-12

### Make these changes and resubmit for another code review.
- Happy coding!

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @aulbytj and @kagaramag :balloon: :lollipop: 

### Thank you for doing the changes requested by the previous reviewer, your project looks amazing, :art: the font work is just brilliant :sparkles:  and I can see you achieve the whole objective of a heat map :100: 

### There is just a couple of requirement left to make :hammer: I'm sorry these changes weren't pointed it out earlier

- [x] It uses `<article>` tag ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/article_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540025/)) Please use this tag instead of using `<div>` tag for all articles(the sections with more text in the content) in your page

- [ ] Your linters aren't working perhaps you have to [activate the repo in stickler](https://stickler-ci.com/)

[OPTIONAL] please add the .vscode folder to your .gitignore

That's it, team! :raised_hands: 

### It was a blast :guitar: to review this project. Please :pray: do these changes :memo:  and submit back for a code review when you are done :rocket: 

Happy Coding! :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

File: index.html [hunk 1]
```diff
@@ -0,0 +1,550 @@
...
+        </p>
+        <button>Learn more about the book</button>
+      </div>
+    </section>
+    <!-- community post -->
+    <section>
+      <h2>community post</h2>
+      <div class="community-post">
+        <div class="rotated"></div>
+        <div>
```
Comment (euqueme) [inline index.html]: line #268 should be `<article>` and not a `<div>`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,550 @@
...
+            <p># Static Generators</p>
+          </div>
+        </article>
+      </div>
+    </section>
+    <!-- latest post -->
+    <section>
+      <h2>latest post</h2>
+      <div class="latest-post bg-gray-1">
+        <div class="md">
```
Comment (euqueme) [inline index.html]: line #95 should be `<article>` and not a `<div>`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,550 @@
...
+          <p>By Rachel Andrew</p>
+
+          <p>
+            SmashingConf is coming to Austin, and in this post, we bring you
+            an interview with one of our speakers, Zach Leatherman, who will
+            be talking about type and font performance.
+          </p>
+        </div>
+
+        <div class="md">
```
Comment (euqueme) [inline index.html]: line #111 should be `<article>` and not a `<div>`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,550 @@
...
+          <p>By Cosima Mielke</p>
+
+          <p>
+            Artists and designers from across the globe once again got out
+            their favorite tools and created unique and inspiring wallpapers
+            to welcome the new month. The wallpapers are available with...
+          </p>
+        </div>
+
+        <div class="md">
```
Comment (euqueme) [inline index.html]: line #125 should be `<article>` and not a `<div>`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,550 @@
...
+            In this article, Vassili Kaplan explains how to use a scripting
+            language to develop cross-platform mobile applications.
+          </p>
+        </div>
+
+        <div class="md">
+          <p>our friends and supporters</p>
+        </div>
+
+        <div class="lg">
```
Comment (euqueme) [inline index.html]: line #144 should be `<article>` and not a `<div>`

File: index.html [hunk 6]
```diff
@@ -0,0 +1,550 @@
...
+          </p>
+
+          <button>Explore features →</button>
+
+          <p>Already have an account? Sign in!</p>
+        </div>
+
+        ​
+
+        <div class="md">
```
Comment (euqueme) [inline index.html]: line #162 should be `<article>` and not a `<div>`

File: index.html [hunk 7]
```diff
@@ -0,0 +1,550 @@
...
+
+          <p>
+            WordPress adoption is massive. So why would a WordPress site
+            consider moving to JAMstack? In this technical case study, we’ll
+            cover what an actual WordPress migration looks like, using
+            Smashing Magazine itself! We’ll talk through the gains and ….
+          </p>
+        </div>
+
+        <div class="md">
```
Comment (euqueme) [inline index.html]: line #180 should be `<article>` and not a `<div>`

File: index.html [hunk 8]
```diff
@@ -0,0 +1,550 @@
...
+
+          <p>
+            We’re taking a look at the brand new version of Microsoft Edge and
+            its Chromium-based rendering engine. What does that change mean
+            for web developers, and how does the death of Microsoft’s own
+            rendering engine impact the web? Drew McLellan talks to …
+          </p>
+        </div>
+
+        <div class="md">
```
Comment (euqueme) [inline index.html]: line #198 should be `<article>` and not a `<div>`

File: index.html [hunk 9]
```diff
@@ -0,0 +1,550 @@
...
+
+          <p>
+            Want to build a website or PWA that attracts a large audience and
+            then converts as much of that audience into paying customers as
+            possible? If that’s the case, what you need to do is use
+            storytelling in your design.
+          </p>
+        </div>
+
+        <div class="md">
```
Comment (euqueme) [inline index.html]: line #213 should be `<article>` and not a `<div>`

File: index.html [hunk 10]
```diff
@@ -0,0 +1,550 @@
...
+
+          <p>
+            How do we know if we are truly impactful as a design team? Are we
+            seen as a vehicle to deliver a solution that moves a needle? It’s
+            time to empower our design teams and give them one voice to show
+            how and when design really adds value.
+          </p>
+        </div>
+
+        <div class="lg">
```
Comment (euqueme) [inline index.html]: line #228 should be `<article>` and not a `<div>`

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

#### Hello again @aulbytj and @kagaramag :wave:  thank you for doing some of the requested changes :handshake:  there is still one mandatory change you have to do to have the approval :memo:  

- [ ] your linters aren't working: I have taken the liberty of looking at other projects you have submitted in the past and as you can see when the linters are working each commit is marked either with an :heavy_check_mark: or an :x: 

- Commit when linters work:
image

- Your last commit:
image

[OPTIONAL] I can see you add some` <article>` tags but you could've added more as I have suggested in your code just now :arrow_double_down:  since this was overlooked by the previous reviewer it's optional but it was in the requirements of the project as I explained in my previous review.

### Please do the mandatory change (Linters have to work) and submit back for a code review when you are done

Any questions? reach to me on slack **@ Maru**

Happy coding! :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

File: index.html (hunk 1 above)
Comment (aulbytj) [reply index.html]: done

**Review by nikasulo — APPROVED** [review]
# Great work incorporating all the changes requested to you by the TSEs
You may merge and keep moving forward :smile: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

File: index.html (hunk 2 above)
Comment (aulbytj) [reply index.html]: done

File: index.html (hunk 4 above)
Comment (aulbytj) [reply index.html]: done

File: index.html (hunk 6 above)
Comment (aulbytj) [reply index.html]: done

File: index.html (hunk 5 above)
Comment (aulbytj) [reply index.html]: done

File: index.html (hunk 7 above)
Comment (aulbytj) [reply index.html]: done

File: index.html (hunk 8 above)
Comment (aulbytj) [reply index.html]: done

File: index.html (hunk 9 above)
Comment (aulbytj) [reply index.html]: done

File: index.html (hunk 10 above)
Comment (aulbytj) [reply index.html]: done

**Pull request merged**